app/views/govuk_publishing_components/components/_attachment.html.erb

```diff
@@ -41,11 +41,11 @@
                 "aria-hidden": true,
                 data: data_attributes do %>
       <% if attachment.document? %>
-        <%= render "govuk_publishing_components/components/attachment/thumbnail_document.svg" %>
+        <%= render "govuk_publishing_components/components/attachment/thumbnail_document" %>
```

Comment:
Out of curiosity, do you know if by renaming the files from `thumbnail_document.svg` to `_thumbnail_document.svg.erb` we would achieve the same result?

Reply:
That hadn't occurred to me, interesting question. I don't know, no, but I think the change to the auditing to only match `/component/name/` and not `/component/partial/partial` is probably good anyway.

Reply:
Might delay merging this to investigate if I can find time.